feeder/config.py

```diff
@@ -42,5 +42,5 @@ class Settings(BaseSettings):
     mqtts_port: int = 8883
     mqtts_public_key: str = "./cert.pem"
     mqtts_private_key: str = "./pkey.pem"
-    http_port: int = 5000
+    http_port: int = 443
```

Comment:
If we do choose to go this route, the daemon itself should not be listening directly on 443. This is a privileged port and would require the daemon to be running as root.

Reply:
We can certainly pick a non privileged port such as 4443. Regarding chrome trusting the self-signed cert, at least on osx, I had to add it to my local keychain and trust it for ssl.